sdk/iothub/azure-resourcemanager-iothub/src/test/java/com/azure/resourcemanager/iothub/generated/IotHubManagerTests.java

```diff
@@ -0,0 +1,161 @@
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Random;
+
+public class IotHubManagerTests extends TestBase {
+
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_WEST2;
+    private String resourceGroupName = "rg" + randomPadding();
+    private IotHubManager iotHubManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        iotHubManager = IotHubManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void testIotHubDescription() {
+        IotHubDescription iotHubDescription = null;
+        try {
+            String descriptionName = "iotHub" + randomPadding();
+
+            Map<String, EventHubProperties> eventHubEndpointsMap = new HashMap<>();
+            eventHubEndpointsMap.put("events", new EventHubProperties()
+                .withRetentionTimeInDays(1L).withPartitionCount(2));
+
+            Map<String, StorageEndpointProperties> storageEndpointsMap = new HashMap<>();
+            storageEndpointsMap.put("$default", new StorageEndpointProperties()
+                .withSasTtlAsIso8601(Duration.ofHours(1L))
+                .withConnectionString(StringUtil.EMPTY_STRING)
+                .withContainerName(StringUtil.EMPTY_STRING));
+
+            Map<String, MessagingEndpointProperties> messagingEndpointsMap = new HashMap<>();
+            messagingEndpointsMap.put("fileNotifications", new MessagingEndpointProperties()
+                .withLockDurationAsIso8601(Duration.ofMinutes(1L))
+                .withTtlAsIso8601(Duration.ofHours(1L))
+                .withMaxDeliveryCount(10));
+
+            // embedmeStart
```

Comment:
sorry, onething more, please move this to the line before those `Map<> = ` code. These are also valuable for user.

Reply:
seems `// @embedme`, not `// embedme`?

Reply:
Ah, yes, good catch. @v-hongli1 Please fix.